public/app/plugins/datasource/loki/components/VariableQueryEditor.tsx

```diff
@@ -70,33 +71,45 @@ export const LokiVariableQueryEditor = ({ onChange, query, datasource }: Props)
   };
 
   return (
-    <InlineFieldRow>
-      <InlineField label="Query type" labelWidth={20}>
-        <Select
-          aria-label="Query type"
-          onChange={onQueryTypeChange}
-          onBlur={handleBlur}
-          value={type}
-          options={variableOptions}
-          width={16}
-        />
-      </InlineField>
+    <>
+      <InlineFieldRow>
+        <InlineField label="Query type" labelWidth={20}>
+          <Select
+            aria-label="Query type"
+            onChange={onQueryTypeChange}
+            onBlur={handleBlur}
+            value={type}
+            options={variableOptions}
+            width={16}
+          />
+        </InlineField>
+        {type === QueryType.LabelValues && (
+          <>
+            <InlineField label="Label" labelWidth={20}>
+              <Select
+                aria-label="Label"
+                onChange={onLabelChange}
+                onBlur={handleBlur}
+                value={{ label: label, value: label }}
+                options={labelOptions}
+                width={16}
+                allowCustomValue
+              />
+            </InlineField>
+          </>
+        )}
+      </InlineFieldRow>
       {type === QueryType.LabelValues && (
-        <>
-          <InlineField label="Label" labelWidth={20}>
-            <Select
-              aria-label="Label"
-              onChange={onLabelChange}
-              onBlur={handleBlur}
-              value={{ label: label, value: label }}
-              options={labelOptions}
-              width={16}
-              allowCustomValue
-            />
-          </InlineField>
+        <InlineFieldRow
```

Comment:
I would remove this and let it take the whole space:

1200px:

![imagen](https://github.com/grafana/grafana/assets/1069378/d32eb069-c496-46ce-a58c-9b0baca2fb69)

1920

![imagen](https://github.com/grafana/grafana/assets/1069378/473ba2a4-3042-491d-8c0f-c2b4e334cd52)

800

![imagen](https://github.com/grafana/grafana/assets/1069378/8fdad278-7400-4538-8e98-32a3f18973ab)

vs with 75%

![imagen](https://github.com/grafana/grafana/assets/1069378/60f53f80-08e1-494b-abd3-4a0534715750)

![imagen](https://github.com/grafana/grafana/assets/1069378/f597a0c3-0e78-45c0-93e7-b7e5123ecf0c)

Reply:
I agree